.github/CODEOWNERS

```diff
@@ -40,6 +40,9 @@
 
 /.github/CODEOWNERS @odoo/rd-code-owners
 
+/addons/account*/ @odoo/rd-accounting
+/addons/l10n_*/ @odoo/rd-accounting
+
 /addons/account/tests/test_portal_attachment.py @odoo/rd-images
```

Comment:
You will also need to adapt this rule to add your team, because it will take precedence over the generic one.